src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -457,6 +457,7 @@ void merge_config_with(ov::AnyMap& lhs, const ov::AnyMap& rhs) {
 struct NPUDesc {
     std::string arch;
     int64_t max_tiles;
+    bool compiler_dq;
```

Comment:
I think it is worth to have default values here, probably even incorrect (e.g., 0 as max_tiles)